go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/im7mortal/kmutex v1.0.1
 	github.com/ipfs/go-cid v0.1.0
 	github.com/ipfs/go-log/v2 v2.3.0
-	github.com/ipld/go-storethehash v0.0.0-20210825183827-efb212b402c8
+	github.com/ipld/go-storethehash v0.0.0-20210915115016-46cc9b178292
```

Comment:
@gammazero noting that this dep is now pointing to the commit before-merge in the upstream